openstf/pipeline/optimize_hyperparameters.py

```diff
@@ -81,7 +81,18 @@ def optimize_hyperparameters_pipeline(
 
     validated_data_with_features = TrainFeatureApplicator(
         horizons=horizons
-    ).add_features(validated_data)
+    ).add_features(validated_data, pj=pj)
+
+    if pj["model"] == "proloaf":
+        # Adds additional proloaf features to the input data, historic_load (equal to the load)
+        if "historic_load" not in list(validated_data_with_features.columns):
+            validated_data_with_features[
```

Comment:
Sonarcloud complains about this part, it is probably better to replace this double if statement with:

if pj["model"] == "proloaf" and "historic_load" not in list(validated_data_with_features.columns):